src/llscan.h

```diff
@@ -0,0 +1,111 @@
+#ifndef SRC_LLSCAN_H_
+#define SRC_LLSCAN_H_
+
+#include <lldb/API/LLDB.h>
+#include <map>
+#include <set>
+
+namespace llnode {
+
+class FindObjectsCmd : public CommandBase {
+ public:
+  ~FindObjectsCmd() override {}
+
+  bool DoExecute(lldb::SBDebugger d, char** cmd,
+                 lldb::SBCommandReturnObject& result) override;
+};
+
+class FindInstancesCmd : public CommandBase {
+ public:
+  ~FindInstancesCmd() override {}
+
+  bool DoExecute(lldb::SBDebugger d, char** cmd,
+                 lldb::SBCommandReturnObject& result) override;
+
+ private:
+  bool detailed_;
+};
+
+class MemoryVisitor {
+ public:
+  virtual ~MemoryVisitor(){};
+
+  virtual uint64_t Visit(uint64_t location, uint64_t available) { return 0; };
```

Comment:
is there any reason to provide such default implementation, instead of doing `= 0` at the end?